data/json/clothing_mods.json

```diff
@@ -28,30 +161,143 @@
-    "flag": "kevlar_padded",
-    "item": "kevlar_plate",
-    "implement_prompt": "Pad with Kevlar",
-    "destroy_prompt": "Destroy Kevlar padding",
+    "id": "tailored_pockets",
+    "flag": "tailored_pockets",
+    "item": "rag",
+    "apply_flags": [ "POCKETS" ],
+    "exclude_flags": [ "SKINTIGHT", "BELTED" ],
+    "valid_parts": [ "TORSO", "LEGS" ],
+    "implement_prompt": "Add pockets",
+    "destroy_prompt": "Remove added pockets",
+    "difficulty": 4,
     "mod_value": [
-      { "type": "bash", "value": 1, "proportion": [ "thickness", "coverage" ] },
-      { "type": "cut", "value": 2, "proportion": [ "thickness", "coverage" ] },
-      { "type": "encumbrance", "value": 2, "round_up": true, "proportion": [ "thickness", "coverage" ] }
+      { "type": "encumbrance", "value": 1, "round_up": true, "proportion": [ "thickness" ] },
+      { "type": "storage", "value": 0.5, "round_up": false }
     ]
   },
   {
     "type": "clothing_mod",
-    "id": "furred",
-    "flag": "furred",
+    "id": "tailored_fur_collar",
+    "flag": "tailored_fur_collar",
     "item": "fur",
-    "implement_prompt": "Line it with fur",
-    "destroy_prompt": "Destroy fur lining",
+    "apply_flags": [ "COLLAR" ],
+    "exclude_flags": [ "SKINTIGHT" ],
+    "valid_parts": [ "TORSO" ],
+    "implement_prompt": "Add a standing fur collar",
+    "destroy_prompt": "Remove added collar",
+    "min_coverage": 80,
+    "difficulty": 3,
+    "mod_value": [ { "type": "warmth", "value": 5, "round_up": true }, { "type": "encumbrance", "value": 2, "round_up": true } ]
+  },
+  {
+    "type": "clothing_mod",
+    "id": "tailored_faux_fur_collar",
+    "flag": "tailored_faux_fur_collar",
+    "item": "faux_fur",
+    "copy-from": "tailored_fur_collar",
+    "implement_prompt": "Add a standing faux fur collar",
+    "destroy_prompt": "Remove added collar"
+  },
+  {
+    "type": "clothing_mod",
+    "id": "tailored_earplugs",
+    "flag": "tailored_earplugs",
+    "item": "ear_plugs",
+    "apply_flags": [ "PARTIAL_DEAF" ],
+    "exclude_flags": [ "DEAF" ],
+    "valid_parts": [ "HEAD" ],
+    "implement_prompt": "Add makeshift hearing protection",
+    "destroy_prompt": "Remove hearing protection",
+    "min_coverage": 60,
+    "no_material_scaling": true,
+    "item_quantity": 1,
+    "difficulty": 1,
+    "mod_value": [ { "type": "encumbrance", "value": 3, "round_up": true } ]
+  },
+  {
+    "type": "clothing_mod",
+    "id": "tailored_long_sleeves",
+    "flag": "tailored_long_sleeves",
+    "item": "rag",
+    "valid_parts": [ "ARMS" ],
+    "exclude_flags": [ "BELTED", "tailored_short_sleeves" ],
+    "implement_prompt": "Lengthen the sleeves",
+    "destroy_prompt": "Restore the sleeves",
+    "min_coverage": 50,
+    "max_coverage": 70,
+    "difficulty": 1,
+    "no_material_scaling": true,
+    "item_quantity": 12,
+    "mod_value": [ { "type": "encumbrance", "value": 3, "round_up": true }, { "type": "warmth", "value": 10, "round_up": true } ]
```

Comment:
Good catch on the coverage mod.

I capped long sleeved at 70% because that seemed to be the upper limit of coverage for items that weren't already explicitly long-sleeved, based on a very unscientific process of spawning a few items I could think of and checking <.<

But it could be dialed in better if you have suggestions. I figure 100% arms coverage = it must already have long sleeves.